src/automation/HISTORY.rst

```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+0.2.0
```

Comment:
Why do we upgrade directly to version `0.2.0` instead of only upgrading a small version?

Reply:
Referring to [version-numbering](https://py-pkgs.org/07-releasing-versioning.html#version-numbering), it is reasonable to upgrade to `0.2.0`.

> Minor release (0.1.0 -> 0.2.0): a minor release typically includes larger bug fixes or new features that are backward compatible, for example, the addition of a new function. It’s fine to have so many minor releases that you need to use two digits (e.g., 0.13.0).